client/login/wp-login/style.scss

```diff
@@ -34,6 +34,15 @@ $image-height: 47px;
 
 .wp-login__main.main {
 	max-width: 400px;
+
+	&.is-jetpack {
+		max-width: 450px;
+		margin-top: calc(96px - 48px);
```

Comment:
Do you think it would be helpful to add a comment on what those values are or put them into a SCSS variable?

Reply:
Added a comment in cf9ca04, thanks!